src/lib/services/data-converter.ts

```diff
@@ -10,6 +10,35 @@ interface WebSocketResponse {
   content: string;
   requestId: string;
 }
+
+export async function convertSinglePayloadDataConverter(
+  payload: Payload,
+  websocket: any,
```

Comment:
Do we have a better type for this or are we using a library that doesn’t have TypeScript types?

Reply:
The latter

Reply:
It'll join `svelte-virtual-list` on my technical debt naughty pile.